### DISCUSSION TITLE
Regenerate Chunks

### DISCUSSION THREAD

**Discussion by m-doescode** [Mojang/minecraft-editor#63] [Feedback]
This would be useful if in the future this is useable on regular worlds as well. But also, this could be still be useful on world editor projects.

If you have made changes to a chunk whether it is via the editor or crosshair mode, but want to revert back to the natural generation, I think it would be useful to have a tool that would allow you to either regenerate chunks or block selections in chunks.

**Comment by ihategravel2** [comment]
Included this in #71
